packages/cactus-plugin-keychain-aws-sm/src/main/typescript/plugin-keychain-vault-aws-sm.ts

```diff
@@ -0,0 +1,267 @@
+import { Server } from "http";
+import { Server as SecureServer } from "https";
+
+import {
+  SharedIniFileCredentials,
+  config as awsConfig,
+  SecretsManager,
+  Credentials,
+} from "aws-sdk";
+
+import { Express } from "express";
+import { Optional } from "typescript-optional";
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+} from "@hyperledger/cactus-common";
+import {
+  ICactusPlugin,
+  ICactusPluginOptions,
+  IPluginWebService,
+  IWebServiceEndpoint,
+  PluginAspect,
+} from "@hyperledger/cactus-core-api";
+
+export enum AwsCredentialType {
+  LocalFile = "FROM_AWS_CREDENTIAL_FILE",
+  InMemory = "FROM_CODE_VARIABLE",
```

Comment:
@jagpreetsinghsasan I'd update the enum values to match the left side of their declaration